src/main/java/twilightforest/block/BlockTFHugeStalk.java

```diff
@@ -30,9 +30,6 @@ public boolean canSustainLeaves(IBlockState state, IBlockAccess world, BlockPos
 	// [VanillaCopy] BlockLog
 	@Override
 	public void breakBlock(World worldIn, BlockPos pos, IBlockState state) {
-		int i = 4;
-		int j = 5;
-
```

Comment:
don't touch [VanillaCopy] methods
(the JIT is going to get rid of these vars anyways)